geniza/corpus/models.py

```diff
@@ -501,6 +501,9 @@ class Document(ModelIndexable, DocumentDateMixin):
     image_order_override = ArrayField(
         models.URLField(), null=True, verbose_name="Image Order", blank=True
     )
+    image_rotation_override = ArrayField(
```

Comment:
Does it have to be an array field? That's the part that feels brittle to me. How does it update when the images are dragged to re-order?

What about using a dictionary in a jsonfield, so you only story the overrides that are specified and they are keyed on an image or canvas id?